### PR TITLE
Add test to ensure newly added Reader labels return correctly

### LINKED ISSUE #12979: VBMS API Update to Support Correct Display of Labels in Reader
## Description
As a user, I would like to see the same label displayed for a document in Reader as users see in VBMS.  For the most part this already exists, however, for some recently added document types (specifically AMA documents) they currently default to 'Other' label in Reader despite having a more detailed label in VBMS.

## Background
Caseflow Reader pulls all documents from VBMS.  Each document has a label in VBMS, such as 'Statement of the Case', 'Notice of Disagreement', 'Rating Decision', etc.  This information comes through the API with VBMS.  Most document labels are pulled but not all are.  Reader displays the label 'Other' whenever it encounters a document for which a label is not pulled.  

When it comes to things that currently display 'Other', of particular importance is AMA documents such as the VA Form 20-0995, VA Form 20-0996, and VA Form 10182.  These documents where added post Reader's launch, hence, I believe, why they all currently display other in Reader while displaying a more detailed name in VBMS.

### PR DESCRIPTION
**Pull request by ajspotts**
Resolves #12979 

### Description
Add test to ensure the labels added in [this](https://github.com/department-of-veterans-affairs/caseflow-commons/pull/170) PR are returning correctly 

### Acceptance Criteria
- [ ] Correct Reader label is pulled in from list stored in [caseflow-commons](https://github.com/department-of-veterans-affairs/caseflow-commons/blob/fb6fa9658825c143eb8d202b87128f34ca7e210b/app/models/caseflow/document_types.rb#L3)

### Testing Plan
1. Run rspec test to check for correct result

### CODE REVIEW

File: spec/models/document_spec.rb [hunk 1]
```diff
@@ -219,6 +219,19 @@
...
+
+    context "when the doc type is a newly recognized label" do 
+      let(:vbms_document) do
+        OpenStruct.new(
+          document_id: "1",
+          doc_type: "1249",
+          alt_doc_types: nil
+        )
+      end
+      it "returns the documents" do
```
Comment (hschallhorn) [inline spec/models/document_spec.rb:231]: I'd reword this to something like "assigns the correct label type"

Comment (ajspotts) [reply spec/models/document_spec.rb:231]: definitely more specific will update thanks!
